index.html

```diff
@@ -87,112 +133,114 @@ <h3>LET'S CONNECT</h3>
         />
         <div class="project-section">
           <h3 class="project-name">Tonic</h3>
-          <div class="project-info">
+          <nav class="project-info">
             <h4 class="project-client">CANOPY</h4>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="programming-language">Back End Dev</span>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="year">2015</span>
-          </div>
+          </nav>
 
           <p class="project-details">
-            A daily selection of privately personalized reads; no accounts or <br />
-            sign-ups required. 
-          </p>  
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
           <ul class="languages">
             <li class="dev-tool">html</li>
             <li class="dev-tool">css</li>
             <li class="dev-tool">javascript</li>
           </ul>
-          <button class="btn" type="button">See Project</button>
+          <button type="button" class="btn">See Project</button>
         </div>
       </div>
 
       <div class="portfolio-border">
         <img
           class="portfolio-image"
           alt="portfolio project"
-          src="Portfolio4.png"
+          src="Portfolio2.png"
         />
-        <div class="project-section">
-          <h2 class="project-name-one">Multi-Post <br />Stories </h2>
-          <div class="project-info-one">
+        <div class="project-section project-section-reverse">
+          <h2 class="project-name">Multi-Post Stories</h2>
+          <nav class="project-info">
             <h4 class="project-client">CANOPY</h4>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="programming-language">Back End Dev</span>
             <img src="Counter.png" class="counter" alt="counter" />
             <span class="year">2015</span>
-          </div>
-          <p class="project-details-one">
-            A daily selection of privately personalized reads; no accounts or<br />
+          </nav>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
             sign-ups required.
           </p>
-          <ul class="languages-one">
+          <ul class="languages">
             <li class="dev-tool">html</li>
             <li class="dev-tool">css</li>
             <li class="dev-tool">javascript</li>
           </ul>
-          <button class="btn-one" type="button">See Project</button>
+          <button type="button" class="btn">See Project</button>
         </div>
       </div>
```

Comment:
[OPTIONAL] The project cards look awesome 🤗 but they would look even better if you can increase the margin around the text area to match the Figma Design as shown below

Your Design:
![image](https://user-images.githubusercontent.com/73714615/186904160-8216210f-b56a-45f8-a8b7-0cd5e529be5e.png)

Figma Design:
![image](https://user-images.githubusercontent.com/73714615/186904266-3026c061-548b-4050-9e23-9f13b07bf8ed.png)